Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.h

```diff
@@ -76,6 +76,10 @@ class RemoteLayerTreeHost {
     CALayer *layerWithIDForTesting(uint64_t) const;
 
     bool replayCGDisplayListsIntoBackingStore() const;
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+    const HashMap<std::pair<WebCore::GraphicsLayer::PlatformLayerID, WebCore::GraphicsLayer::PlatformLayerID>, CGRect>& overlayRegionsWithIDs() const { return m_overlayRegionsWithIDs; }
```

Comment:
now that this is in the RemoteLayerTreeHost I think the key can be a layerID instead of a pair of IDs